avocado/plugins/runner_nrunner.py

```diff
@@ -69,12 +71,14 @@ def initialize(self):
 
         help_msg = (
             'URI where status server will listen on. Usually a "HOST:PORT" '
-            'string. This is only effective if "status_server_auto" is disabled'
+            'string. This is only effective if "status_server_auto" is disabled. '
+            'If "status_server_uri" is not set. Value from "status_server_listen " '
```

Comment:
I believe this should read:

'If "status_server_uri" is not set, the value from "status_server_listen " '